src/include/crowbar/complex.rb

```diff
@@ -644,8 +644,8 @@ def HandleLocationURL(key, event)
       @repos.each do |platform, arches|
         arches.each do |arch, repos|
           distro = case platform
-          when "suse-12.2"
-            "sles12-sp2-#{arch}"
+          when "suse-12.4"
```

Comment:
Do we actually need such complicated platform handling when there is actually only a single platform to use? 

Also maybe the platform name should be a constant, this would avoid repeating the same string at many places. Ideally we should need to change only a single place in the code, this is fragile as we might overlook some place...

Reply:
You are right, it's just that this module is not much used anyway, and is gonna be dropped after SOC9. So I do not want to invest in code cleanup...

Reply:
If the plan is to drop this module in the future then OK.